src/gluonts/shell/serve/__init__.py

```diff
@@ -0,0 +1,169 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Can we maybe put this in another file and just import the stuff we want to expose here?
This has been our convention so far regarding `__init__.py`.